fpga_interchange/add_prim_lib.py

```diff
@@ -86,33 +86,30 @@ def main():
                 if upto:
                     start, end = end, start
 
-                cell.add_bus_port(
-                    name=port_name,
-                    direction=direction,
-                    start=start,
-                    end=end,
-                    property_map=property_map)
+                cell.add_bus_port(name=port_name,
+                                  direction=direction,
+                                  start=start,
+                                  end=end,
+                                  property_map=property_map)
             else:
-                cell.add_port(
-                    name=port_name,
-                    direction=direction,
-                    property_map=property_map)
+                cell.add_port(name=port_name,
+                              direction=direction,
+                              property_map=property_map)
         prim_lib.add_cell(cell)
 
     libraries = {}
     libraries[args.library] = prim_lib
     # Create the netlist
-    netlist = LogicalNetlist(
-        name=args.library,
-        property_map={},
-        top_instance_name=None,
-        top_instance=None,
-        libraries=libraries)
+    netlist = LogicalNetlist(name=args.library,
```

Comment:
Is there a reason for what appear to be just format changes in existing code that should have been auto-formatted already?